core/domain/prod_validation_jobs_one_off.py

```diff
@@ -30,9 +29,21 @@ class BaseModelValidator(object):
     """Base class for validating models."""
 
     errors = {}
+    # external_models is a map that's keyed by field name. Each value consists
+    # of the model class and a list of (external_key, external_model) tuples.
+    external_models = {}
 
     @classmethod
-    def _external_id_relationships(cls):
+    def _validate(cls, item):
```

Comment:
Just to check, do you always expect this to be a list of functions that's just executed in order?

If so, is it worth making it declarative, e.g.

    validation_functions = [
        name_of_func1,
        name_of_func2
    ]

and then doing something like

    for func in validation_functions:
        func(item)

Reply:
Yes, declarative makes sense.